tools/get_dicom_files.pl

```diff
@@ -90,7 +95,9 @@ =head2 Methods
     ["-d", "string", 1,   \$tmpExtractBaseDir, 
         "base path of the temporary directory where files are extracted"],   
     ["-o", "string", 1,   \$outTarBasename, 
-        "base path of the final .tar.gz file to produce (defaults to 'dicoms')"],   
+        "base path of the final .tar.gz file to produce (defaults to 'dicoms')"], 
+    ["-p", "const" , 1,   \$usePSCID, 
+        "include the PSCID in the components of the archived DICOM file paths"]
```

Comment:
components? Maybe replace by filenames instead?

Reply:
not entirely sure what you mean by components here

Reply:
Updated the description.